app/assets/stylesheets/success_stories.css.scss

```diff
@@ -0,0 +1,25 @@
+.success-stories-photo {
+  border-radius: 50%;
+  height: 120px;
+  width: 120px;
+}
+
+.success-stories-photo--left {
+  margin-left: 15px;
+}
+
+.success-stories-photo--right {
```

Comment:
@jjhampton excellent! 

> On Sep 11, 2016, at 10:37 PM, John Hampton <notifications@github.com> wrote:
> 
> In app/assets/stylesheets/success_stories.css.scss:
> 
> > @@ -0,0 +1,25 @@
> > +.success-stories-photo {
> > +  border-radius: 50%;
> > +  height: 120px;
> > +  width: 120px;
> > +}
> > +
> > +.success-stories-photo--left {
> > +  margin-left: 15px;
> > +}
> > +
> > +.success-stories-photo--right {
> @rickr I went ahead and just tore out those modifier classes (--left & --right) - no need to have them once I added in the nth-of-type CSS rules you suggested.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>